src/main/app/case/case.ts

```diff
@@ -507,6 +507,10 @@ export interface Case {
   childFirstName?: ChildrenDetails['firstName'];
   childLastName?: ChildrenDetails['lastName'];
   cd_children?: ChildrenDetails[];
+  otherChildFirstName?: ChildrenDetails['firstName'];
+  otherChildLastName?: ChildrenDetails['lastName'];
+  cd_otherChildren?: ChildrenDetails[];
```

Comment:
You cannot simply use childrenDetails interface . Please extend the base interface and make sure you have only required properties for other children.

Reply:
Agree with Vivek.
@ShakiraKP Can we please reuse childFirstName & childLastName for other children as well.
Anyways it's only for validation, after that data will be added into respective field cd_otherChildren